modules/builds-source-code.adoc

```diff
@@ -23,15 +23,15 @@ ifndef::openshift-online[]
   dockerfile: "FROM openshift/ruby-22-centos7\nUSER example" <3>
 endif::[]
 ----
-<1> The `git` field contains the URI to the remote Git repository of the source code. Optionally, specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name.
+<1> The `git` field contains the `URI` to the remote Git repository of the source code. Specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name, if not specified "master" will be picked as the default value.
```

Comment:
```suggestion
<1> The `git` field contains the Uniform Resource Identifier (URI) to the remote Git repository of the source code. You must specify the value of the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name. The default value of the `ref` field is `main`.
```

Reply:
@shipsing we would need to stick by `master` in the last sentence here, i.e., 
```
The default value of the `ref` field is `main`
```
By default builder picks up the `master` branch even when default branch for the project is `main`. In such scenarios, it becomes mandatory to specify `ref: main` in the buildconfig.
Details are available here: https://issues.redhat.com/browse/OCPBUGS-13977

I hope this helps with the context. Please let me know your inputs.